tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
   "compilerOptions": {
     "module": "esnext",
-    "target": "esnext",
+    "target": "es2019",
+    "lib": ["ES2019", "DOM"],
```

Comment:
Why has this worked before without `"DOM"` in `"lib"`, and why do we need to add `"DOM"` now?

Reply:
We didn't have a "lib" key at all before. This now specifies the libraries that TypeScript knows we have, so ES2019's and the DOM's. I'm not sure if it's necessary, but the official TypeScript page mentions it (see commit message)